lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -165,47 +165,4 @@ private static function remove_JSON_comments( $array ) {
 		return $array;
 	}
 
-	/**
-	 * Returns the data merged from multiple origins.
-	 *
-	 * There are three sources of data (origins) for a site:
-	 * default, theme, and custom. The custom's has higher priority
-	 * than the theme's, and the theme's higher than default's.
-	 *
-	 * Unlike the getters {@link get_core_data},
-	 * {@link get_theme_data}, and {@link get_user_data},
-	 * this method returns data after it has been merged
-	 * with the previous origins. This means that if the same piece of data
-	 * is declared in different origins (user, theme, and core),
-	 * the last origin overrides the previous.
-	 *
-	 * For example, if the user has set a background color
-	 * for the paragraph block, and the theme has done it as well,
-	 * the user preference wins.
-	 *
-	 * @since 5.8.0
-	 * @since 5.9.0 Added user data, removed the `$settings` parameter,
-	 *              added the `$origin` parameter.
-	 *
-	 * @param string $origin Optional. To what level should we merge data.
-	 *                       Valid values are 'theme' or 'custom'. Default 'custom'.
-	 * @return WP_Theme_JSON
-	 */
-	public static function get_merged_data( $origin = 'custom' ) {
```

Comment:
I don't know why this method was part of `lib/experimental`. There are a lot of things there that shouldn't. I'm going to prepare a different PR to fix how these classes are organized because there is a lot of confusion around where code should live.